src/status_im/chat/views/input/send_button.cljs

```diff
@@ -15,6 +15,9 @@
        (animation/timing spin-value {:toValue  to-spin-value
                                      :duration 300})))))
 
+(defn sendable? [input-text]
+  (not (or (string/blank? input-text) (zero? (compare input-text "/")))))
```

Comment:
Why `(zero? (compare s)))` and not equals?

Reply:
@jeluard You're right, should use equals. Now fixed! :)